docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
+is biased when the risk factors are correlated.**
 
-Dividing through and cancelling terms yields a solution:
+The true population attributable fraction for the combined effect of two risk factors,
+:math:`r1` and :math:`r2`, on an outcome :math:`O` such as a mortality rate is (assuming no effect modification, as we typically do):
 
 .. math::
-
-
-   \frac{1}{1-\text{PAF}} = \int_{e_1} \int_{e_2} \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-   p_{e_1,e_2} de_1 de_2,
-
-.. code:: python
-
-    # there are lots of ways to approximate this integral, but here is a cool one:
-    rr_1 = 1.5
-    rr_2 = 3
-
-    e1 = np.clip((df['SBP'] - 120)/10, 0, np.inf)  # rescale exposures
-    e2 = np.clip((df['LDL-C'] - 4), 0, np.inf)
-
-    one_over_one_minus_paf = np.mean(rr_1**e1 * rr_2**e2)
-    paf = 1 - 1/(one_over_one_minus_paf)
-    paf
-
-
-
-
-.. parsed-literal::
-
-    0.936944660107624
-
-
-
-.. code:: python
-
-    # how does this compare to multiplicative approx of paf?
-    ooomp_1 = np.mean(rr_1**e1)
-    paf_1 = 1 - 1/ooomp_1
-
-    ooomp_2 = np.mean(rr_2**e2)
-    paf_2 = 1 - 1/ooomp_2
-
-    1 - (1 - paf_1) * (1 - paf_2)
-
-
-
-
-.. parsed-literal::
-
-    0.8708098034918508
-
-
-
-These values are somewhat different, reflecting the impact of the
-correlation in SBP and LDL-C in our simulated data. If we used the
-*multiplicative* PAF, our estimated rate would be quite different. Let’s
-look at a simpler model of categorical risks.
-
-Here is an example of categorial risk exposures
-=======================================================
-
-.. code:: python
-
-    # simulate categorical data (to be replaced with real data, e.g. from MALED/Surveys, eventually)
-    df = pd.DataFrame(index=range(N))
-    # Uncorrelated, binomial
-    df['bin_a'] = np.random.binomial(1, 0.3, size=N)
-    df['bin_b'] = np.random.binomial(1, 0.1, size=N)
-
-    rr_1 = 2.5
-    rr_2 = 5
-
-
-
-Define what we will call a joint PAF:
+  O = O_{tmrel} * RR_{r1} * RR_{r2}
 
 .. math::
+  PAF = \frac{E(O) - E(O_{tmrel})}{E(O)} = 1 - \frac{O_{tmrel}}{E(O_{tmrel} * RR_{r1} * RR_{r2})} = 1 - \frac{1}{E(RR_{r1} * RR_{r2})}
 
+where :math:`O_{tmrel}` is a constant representing the outcome (rate) among those at the
+theoretical minimum risk exposure level.
 
-   PAF_{joint} = 1 - \frac{1}{\frac{1}{n}\sum_{i=1}^{n} RR_1^{e1_i} \cdot RR_2^{e2_i}}
+Default approach
+^^^^^^^^^^^^^^^^
 
-And also what we will call a multiplicative PAF:
+By default in Vivarium, a joint PAF is calculated with the "multiplicative" approach:
 
 .. math::
+	PAF_{multiplicative} = 1 - (1 - PAF_{r1}) \cdot (1 - PAF_{r2}) = 1 - \frac{1}{E(RR_{r1})} \cdot \frac{1}{E(RR_{r2})}
 
+When :math:`RR_{r1}` and :math:`RR_{r2}` are independent, :math:`E(RR_{r1}) * E(RR_{r2}) = E(RR_{r1} * RR_{r2})`
+so :math:`PAF = PAF_{multiplicative}`; this is not the case when the risks are correlated.
+The bias gets larger the more correlated the risks are, and the larger the true PAF is.
```

Comment:
We could link to James's .pdf writeup computing the error as a function of the PAF.
[Why_is_getting_the_paf_right_.pdf](https://github.com/ihmeuw/vivarium_research/files/10273724/Why_is_getting_the_paf_right_.pdf)

Reply:
And maybe we should ask @collijk for the source LaTeX as well...